README.md

```diff
@@ -389,7 +389,8 @@ node-fetch also supports spec-compliant FormData implementations such as [formda
 
 ```js
 import fetch from 'node-fetch';
-import FormData from 'formdata-node';
+import {FormData} from 'formdata-polyfill/esm-min.js';
+import {FormData} from 'formdata-node';
```

Comment:
Should both of these be here? 🤔

Reply:
They should both work, you can pick one or the other